content/Api/Blip.Api.Template.Tests/Blip.Api.Template.Tests.csproj

```diff
@@ -1,18 +1,21 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <ProjectGuid>sonarqube-guid-tests</ProjectGuid>
+    <ProjectGuid>{FE7A5C89-B282-4526-A91B-45A586FA9BD7}</ProjectGuid>
```

Comment:
whenever you build the project, or open in VS even, the placeholder gets replaced by the generated GUID. Revert back to the placeholder